espnet/nets/pytorch_backend/transformer/positionwise_feed_forward.py

```diff
@@ -10,3 +10,17 @@ def __init__(self, d_model, d_ff, dropout):
 
     def forward(self, x):
         return self.w_2(self.dropout(torch.relu(self.w_1(x))))
+
+
+class PositionwiseConv1d(torch.nn.Module):
+    def __init__(self, d_model, d_ff, kernel_size, dropout):
+        super(PositionwiseConv1d, self).__init__()
+        self.w_1 = torch.nn.Conv1d(d_model, d_ff, kernel_size,
+                                   stride=1, padding=(kernel_size - 1) // 2)
+        self.w_2 = torch.nn.Conv1d(d_ff, d_model, kernel_size,
+                                   stride=1, padding=(kernel_size - 1) // 2)
+        self.dropout = torch.nn.Dropout(dropout)
+
+    def forward(self, x):
+        x = torch.relu(self.w_1(x.transpose(-1, 1))).transpose(-1, 1)
+        return self.w_2(self.dropout(x).transpose(-1, 1)).transpose(-1, 1)
```

Comment:
Why don't you apply dropout to `self.W_2`?

Reply:
I followed your FFN implementation.
Is there any reason?

Reply:
Oh, it is for no reason :fearful: 

I should have applied dropout inside there instead of outside ... https://github.com/espnet/espnet/blob/master/espnet/nets/pytorch_backend/transformer/decoder_layer.py

Reply:
Maybe we can keep it for the later PR.

Reply:
Let me confirm.
linear -> relu -> dropout -> linear -> dropout ?

Reply:
> Maybe we can keep it for the later PR.

OK.